docs/_documentations/remotedeploy-vscode.md

```diff
@@ -44,7 +44,9 @@ The IDE and Codewind are connected.
 
 # 2. Add a deployment registry
 
-Before projects can be deployed on Kubernetes, you need to specify a Docker Registry. Docker Hub is used in this example: 
+Before projects can be deployed on Kubernetes, you need to specify a Docker registry.
```

Comment:
I think this should be `Docker push registry` too, and same comment about Codewind style projects.